src/routes/organizers.jsx

```diff
@@ -145,6 +128,14 @@ const organizers = [
     discourseUrl: "https://discourse.nixos.org/u/picnoir",
     responsibilities: ["A/V"]
   },
+  {
+    name: "Ral",
+    // github: "",
+    // githubUrl: "https://github.com/",
```

Comment:
Don't know the handle

Reply:
For now this is fine: https://matrix.to/#/!GlDwnQuDPMNzaUXCJB:lassul.us/$UykofY2UVRQuRzc2QI_LYFBCy_BA1yKzrMnCxJ__3o4?via=lassul.us&via=matrix.org&via=nixos.dev